cli-backup/src/main/scala/io/aiven/guardian/kafka/backup/Main.scala

```diff
@@ -73,7 +73,10 @@ class Entry(val initializedApp: AtomicReference[Option[App[_]]] = new AtomicRefe
                   block.withBootstrapServers(value.toList.mkString(","))
 
               Some(block).validNel
-            case None if Options.checkConfigKeyIsDefined("kafka-client.bootstrap.servers") => None.validNel
+            case None
```

Comment:
This is a quick fix, there are multiple configurations, a global one which is shared with `kafka-client.bootstrap.servers` and a specific one just for the consumer at `akka.kafka.consumer.kafka-clients.bootstrap.servers`